src/main/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicy.java

```diff
@@ -120,7 +125,7 @@ public ChargeableFee getActualCostFee() {
   }
 
   public boolean shouldRefundFees(DateTime lostDateTime) {
-    return feeRefundInterval == null
+    return feeRefundInterval.isEmpty()
       || feeRefundInterval.hasNotPassedSinceDateTillNow(lostDateTime)
       || feeRefundInterval.isEqualToDateTillNow(lostDateTime);
```

Comment:
This suggests that refunds should be issued if no refund interval is defined, is that the case?

Reply:
Yes, if the refund is not defined or it is empty than refund is allowed any time. Otherwise, only within the defined period after declare lost. 

![image](https://user-images.githubusercontent.com/9094650/90506795-7e931d00-e15d-11ea-8991-4c642289574c.png)

Reply:
Ok, thanks.

I'm wondering if it makes sense to make it clear that the check for zero duration is really a proxy for no interval being defined.